### ISSUE TITLE
FSNN-7 (DST) and Wide- 7B-DSNN implementation details

### ISSUE THREAD

**Issue by alexandercantrell** [iCGY96/AAP#1]
I noticed you had the FSNN and DSNN implementation for ResNet added, but not the FSNN-7 (DST) and Wide-7B-DSNN implementations. Would you be able to post these implementations or go a bit more into how they were structured so  I can implement them myself? Thanks!

**Comment by gary23ai** [comment]
Kindly consult the implementation detailed in this repository: [https://github.com/fangwei123456/Spike-Element-Wise-ResNet]. We plan to update the code subsequent to the official publication of our paper.

**Comment by alexandercantrell** [comment]
Yes, I was working off of that repository, but was unable to achieve the same results as in the paper so I had a few things I was hoping to verify. 

1. For Wide-7B-DSNN on Cifar10-DVS was the AAC detached during testing or was it kept on? 
2. How do you account for the MaxPooling operation in the AAC? Is it applied to both outputs?
3. For downsampling in your DSNN file you use two convolution operations sequentially when the stride is > 1. Is this also done in these two implementations or is normal convolution applied as the stride isn't ever > 1 due to MaxPooling?
4. 
Sorry for all the questions and thank you in advance for your help!
